ros/src/common/libs/state_machine_lib/include/state_machine_lib/state.hpp

```diff
@@ -91,9 +91,9 @@ class State
     CallbackExitFunc = _f;
   }
 
-  void showStateName(void)
+  std::string getStateName(void)
```

Comment:
Returning a const reference here should be slighty faster and it's safer to declare the method as const:

```const std::string & getStateName(void) const```